### PR TITLE
Run install off the same command

### PR DESCRIPTION
**Pull request by Itxaka**
Changes the uki install to be run with the same command as the normal install so it can leverage the normal workflow.

autodetects if is under uki by checking the sentinel set by immucore and the cmdline

Part of https://github.com/kairos-io/kairos/issues/2049

### CODE REVIEW

File: internal/agent/install.go [hunk 1]
```diff
@@ -217,66 +217,90 @@ func RunInstall(c *config.Config) error {
...
 	}
 
-	// Load the installation spec from the Config
-	installSpec, err := config.ReadInstallSpecFromConfig(c)
-	if err != nil {
-		return err
-	}
+	// UKI path. Check if we are on UKI AND if we are running off a cd, otherwise it makes no sense to run the install
+	// From the installed system
+	if internalutils.IsUki() && internalutils.UkiBootMode() == internalutils.UkiRemovableMedia {
```
Comment (mauromorales) [inline internal/agent/install.go:222]: can we introduce two different functions for the control flow instead of putting it all in this function?

Comment (Itxaka) [reply internal/agent/install.go:222]: can you rephrase that? I dont get youo you are suggesting sorry. 

Also, this should be removed and only the uki boot mode one used as that internally cheks if its on uki first.

Comment (mauromorales) [reply internal/agent/install.go]: sorry my bad, I forget ppl are not in my head LOL

I felt that it would be better to have the uki and non-uki process put in different functions, because the outer function was already big enough and now it increased

if it will be removed soon then no need to change

**Review by mauromorales — APPROVED** [review]

File: internal/agent/install.go (hunk 1 above)
Comment (Itxaka) [reply internal/agent/install.go]: ahh I see!

Will do

Comment (Itxaka) [reply internal/agent/install.go]: nicer?

**Review by jimmykarily — APPROVED** [review]

**Review by mauromorales — APPROVED** [review]
